### PR TITLE
Fuzz testing fixes for found problems

### PR DESCRIPTION
**Pull request by jeffallen**
Fixes dedis/onet#293.

### CODE REVIEW

File: decode.go [hunk 1]
```diff
@@ -102,7 +105,7 @@ func (de *decoder) message(buf []byte, sval reflect.Value) error {
 		rem, err := de.value(wiretype, buf, field)
 		if err != nil {
 			if fieldi < len(fields) && fields[fieldi] != nil {
-				return fmt.Errorf("Error while decding FieldName %s: %v", fields[fieldi].Name, err)
+				return fmt.Errorf("Error while decoding FieldName %s: %v", fields[fieldi].Name, err)
```
Comment (ineiti) [inline decode.go]: HERE IT IS! I thought I fixed it, but didn't find it. I'll be happy to read `Error while decOding FieldName`. Can you make it a capital o? No, joke only...

File: fuzz.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
+// To fuzz-test this package:
+//
+//   $ go get github.com/dvyukov/go-fuzz/go-fuzz
+//   $ go get github.com/dvyukov/go-fuzz/go-fuzz-build
+//   $ go-fuzz-build github.com/dedis/protobuf
+//   $ go-fuzz -workdir=workdir -bin protobuf-fuzz.zip
```
Comment (kc1212) [inline fuzz.go]: It is worthwhile putting the fuzzy tests on travis too?

**Comment by ineiti** [comment]
LGTM

File: fuzz.go (hunk 2 above)
Comment (ineiti) [reply fuzz.go]: Please don't - having more flakyness is not what we need...

Comment (jeffallen) [reply fuzz.go]: No, they are long, in fact infinite running. So you run them while actively working on the parser only. I will teach a workshop on them sometime, it's really fascinating technology.

**Pull request merged**